### PR TITLE
tc struct fields addition

### PR DESCRIPTION
**Pull request by Sosutha**
* Addition of fields in table key value structure

* Initialization of key structure

* Updated testcase outputs

https://github.com/p4tc-dev/p4tc-e2e/issues/58
https://github.com/p4tc-dev/p4tc-e2e/issues/57

### CODE REVIEW

File: backends/tc/runtime/pna.h [hunk 1]
```diff
@@ -105,6 +105,9 @@ struct p4tc_table_entry_act_bpf_params__local {
 
 struct __attribute__((__packed__)) p4tc_table_entry_act_bpf {
         u32 act_id;
+        u32 hit:1,
```
Comment (usha1830) [inline backends/tc/runtime/pna.h:108]: Indentation is off

**Review by usha1830 — APPROVED** [review]
Looks fine to me

File: backends/tc/runtime/pna.h (hunk 1 above)
Comment (Sosutha) [reply backends/tc/runtime/pna.h:108]: Corrected

**Review by vbnogueira — APPROVED** [review]

**Review by psivanup — APPROVED** [review]
Looks fine

**Pull request closed without merging**